.ci/aarch64/kubernetes/init.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+flannel_config="https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
```

Comment:
This looks rather magic. I'm assuming this weave site parses the version output and returns a suitable config, but I wonder if we should log `kubectl version` in case it gets it wrong?

Reply:
yep, I got this magic command [here] in there docs(https://www.weave.works/blog/weave-net-kubernetes-integration/). 
I‘m with you the log part. but this is how kubectl version looks like in my host
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.1-dirty", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"dirty", BuildDate:"2019-06-18T09:27:34Z", GoVersion:"go1.12.6", Compiler:"gc", Platform:"linux/arm64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-08T17:02:58Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/arm64"}

```
How do I tell if it is wrong? 😖

Reply:
Hi @Pennyzct - sorry, all I mean is let's log `kubectl version` (and maybe `$flannel_config`) so that *if* the test fails, that might give us a clue if the failure was somehow caused by the flannel config.

Reply:
the name of the variable is `flannel_config` but it points to wave,  this can be prone to confusions and errors , isn't it ?

Reply:
@devimc yep, with you, we should re-name a generic name for it. 👻